templates/taxi/car_list.html

```diff
@@ -1,17 +1,46 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>
+    Car list
+    <a class="btn btn-primary" style="float: right" href="{% url "taxi:car-create" %}">Create</a>
+  </h1>
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <thead>
+      <tr>
+        <th scope="col">Id</th>
+        <th scope="col">Manufacture</th>
+        <th scope="col">Model</th>
+        <th scope="col">Country</th>
+        <th scope="col">Update</th>
+        <th scope="col">Delete</th>
+      </tr>
+      </thead>
+      <tbody>
```

Comment:
I think our task is clearly defined
>
2. On the car list page add button that leads to the car creation page.
**On the car detail page add buttons that lead to the car update page and car deletion page.**